libs/cardano-ledger-test/src/Test/Cardano/Ledger/STS.hs

```diff
@@ -171,34 +190,34 @@ prop_RATIFY =
 prop_CERT :: Property
 prop_CERT =
   stsPropertyV2 @"CERT" @ConwayFn
-    certEnvSpec
-    (\_env -> certStateSpecEx)
-    (\env st -> txCertSpec env st)
+    (certEnvSpec conwayWitUniv)
+    (\_env -> certStateSpec conwayWitUniv conwayDelegatees)
+    (\env st -> conwayTxCertSpec conwayWitUniv env st)
```

Comment:
Since this is a property, you can also use `forAll` from the `QuickCheck` library to generate the universe. That way you won't have to use `unsafePerformIO`

Reply:
I didn't know you could use it like that. That is actually very cool. Evertything is so much cleaner.